packages/wallet/src/services/HandlesTracker.ts

```diff
@@ -135,6 +135,7 @@ export const createHandlesTracker = (
           }
           return {
             ...assetInfo,
+            addresses: { cardano: txOut.address },
```

Comment:
Bitcoin address might be available in the datum. Not sure if we need it here though.

Reply:
This task is propaedeutic for the dismissal of this service, I changed it just to make it compliant with the core type change.